test/integration_test/applicationbackup_test.go

```diff
@@ -1053,3 +1059,16 @@ func deleteApplicationRestoreList(appRestoreList []*storkv1.ApplicationRestore)
 	logrus.Info("Deleted all restores")
 	return nil
 }
+
+func getBackupLocationForVolumeDriver(volumeDriver string) (storkv1.BackupLocationType, error) {
```

Comment:
Good for now, but we should put this in the driver in a separate change

Reply:
will send separate PR in torpedo